app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        @rents = Rent.all
+        render json: @rents
+      end
+
+      def create
+        @rent = Rent.new(rent_params)
+        if @rent.save
+          render json: @rent
+        else
+          render status: 500
```

Comment:
This is quite extreme :P. when you run the `save` method, 2 things happens, first the `valid?` method is executed, that one goes through all your validations and add `errors` to an array for each one of them that don't pass. At the end of this, if the array of errors is empty, it performs a save into the database and returns `true`, and if not , it returns `false` and leaves the `errors` for us to know what prevents that model from being save. You can access to that executing `@rent.errors`